pkg/command/login_test.go

```diff
@@ -11,61 +11,91 @@ import (
 
 func TestPrepareTanzuContextName(t *testing.T) {
 	testCases := []struct {
-		orgName   string
-		endpoint  string
-		isStaging bool
-		forceCSP  bool
-		expected  string
+		orgName       string
+		ucpEndpoint   string
+		endpoint      string
+		saasEndpoints []string
+		isStaging     bool
+		forceCSP      bool
+		expected      string
 	}{
 		// Test case for normal input with no staging environment and default endpoint.
 		{
-			orgName:   "MyOrg",
-			endpoint:  centralconfig.DefaultTanzuPlatformEndpoint,
-			isStaging: false,
-			expected:  "MyOrg",
+			orgName:       "MyOrg",
+			ucpEndpoint:   centralconfig.DefaultTanzuPlatformEndpoint,
+			endpoint:      centralconfig.DefaultTanzuPlatformEndpoint,
+			saasEndpoints: []string{centralconfig.DefaultTanzuPlatformEndpoint},
+			isStaging:     false,
+			expected:      "MyOrg",
 		},
 		// Test case for normal input with staging environment and default endpoint.
 		{
-			orgName:   "MyOrg",
-			endpoint:  centralconfig.DefaultTanzuPlatformEndpoint,
-			isStaging: true,
-			expected:  "MyOrg-staging",
+			orgName:       "MyOrg",
+			ucpEndpoint:   centralconfig.DefaultTanzuPlatformEndpoint,
+			endpoint:      centralconfig.DefaultTanzuPlatformEndpoint,
+			saasEndpoints: []string{centralconfig.DefaultTanzuPlatformEndpoint},
+			isStaging:     true,
+			expected:      "MyOrg-staging",
 		},
-		// Test case for normal input with no staging environment and custom SaaS endpoint.
+		// Test case for normal input with no staging environment and custom endpoint with force CSP.
 		{
-			orgName:   "MyOrg",
-			endpoint:  "https://custom-endpoint.com",
-			isStaging: false,
-			expected:  "MyOrg-86fd8133",
-			forceCSP:  true,
+			orgName:       "MyOrg",
+			ucpEndpoint:   "https://ucp.custom-endpoint.com",
+			endpoint:      "https://custom-endpoint.com",
+			saasEndpoints: []string{centralconfig.DefaultTanzuPlatformEndpoint},
+			isStaging:     false,
+			expected:      "MyOrg-70217fc3",
+			forceCSP:      true,
 		},
-		// Test case for normal input with staging environment and custom SaaS endpoint.
+		// Test case for normal input with staging environment and custom endpoint with force CSP.
 		{
-			orgName:   "MyOrg",
-			endpoint:  "https://custom-endpoint.com",
-			isStaging: true,
-			expected:  "MyOrg-staging-86fd8133",
-			forceCSP:  true,
+			orgName:       "MyOrg",
+			ucpEndpoint:   "https://ucp.custom-endpoint.com",
+			endpoint:      "https://custom-endpoint.com",
+			saasEndpoints: []string{centralconfig.DefaultTanzuPlatformEndpoint},
+			isStaging:     true,
+			expected:      "MyOrg-staging-70217fc3",
+			forceCSP:      true,
+		},
+		// Test case for normal input with new ucpEndpoint and platform endpoints which is actually a SaaS endpoints
```

Comment:
nit: which is actually a SaaS endpoints -> which are actually SaaS endpoints. Same for L68

Reply:
Thanks. Updated.